app.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
It's a declarative way of defining settings, add-ons and environment variables for Heroku. I believe if this file is defined correctly, when you come to clone/fork an application, all the add-ons will be cloned as well.